### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by davidbuzinski**

### CODE REVIEW

**Comment by mw-hrastega** [comment]
> _ No description provided. _

Should README.md also be part of this pull request?

**Comment by davidbuzinski** [comment]
> > _ No description provided. _
> 
> Should README.md also be part of this pull request?

Yes, it's included. I just updated the versions in the examples.

File: README.md [hunk 1]
```diff
@@ -26,25 +26,39 @@ Each of these pipeline definitions does four things:
...
 steps:
-  - task: InstallMATLAB@0
-  - task: RunMATLABTests@0
+  - task: InstallMATLAB@1
+  - task: RunMATLABTests@1
     inputs:
       sourceFolder: source
       codeCoverageCobertura: coverage.xml
-  - script: bash <(curl -s https://codecov.io/bash)
+  - script: |
```
Comment (mw-hrastega) [inline README.md:33]: The original code was terse. This replacement is quite verbose and hard to understand. Can I know why we had to make this replacement?

File: README.md [hunk 2]
```diff
@@ -26,25 +26,39 @@ Each of these pipeline definitions does four things:
...
+      curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step  
+      curl -Os https://cli.codecov.io/latest/linux/codecov
+      curl -Os https://cli.codecov.io/latest/linux/codecov.SHA256SUM
+      curl -Os https://cli.codecov.io/latest/linux/codecov.SHA256SUM.sig
+      gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
+
+      # upload to Codecov 
+      shasum -a 256 -c codecov.SHA256SUM
+      sudo chmod +x codecov
+      ./codecov upload-process -t $(CODECOV_TOKEN)
```
Comment (mcafaro) [inline README.md:48]: Why not use the codecov action? https://github.com/marketplace/actions/codecov

**Comment by mw-hrastega** [comment]
> > > _ No description provided. _
> > 
> > 
> > Should README.md also be part of this pull request?
> 
> Yes, it's included. I just updated the versions in the examples.

Ok, I think some text also needs to be updated:

1) You wanted to discontinue support for Travis CI. Does it still make sense to have it on the page?
2) The Caveats section is outdated.

File: README.md (hunk 1 above)
Comment (davidbuzinski) [reply README.md:33]: It looks like the bash script is still available, but they are moving towards using their CLI going forward. When you setup a new project on codecov.io and choose "Other CI" (Not GitHub Actions or CircleCI) they recommend doing things this way
Screenshot 2024-04-02 at 11 10 25 AM
mend this code:

**Comment by davidbuzinski** [comment]
> > > > _ No description provided. _
> > > 
> > > 
> > > Should README.md also be part of this pull request?
> > 
> > 
> > Yes, it's included. I just updated the versions in the examples.
> 
> Ok, I think some text also needs to be updated:
> 
>     1. You wanted to discontinue support for Travis CI. Does it still make sense to have it on the page?
> 
>     2. The Caveats section is outdated.

Good idea. I removed the Travis example and caveats section.

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md:33]: Good to know. Thanks for clarifying, David.

**Review by mw-hrastega — APPROVED** [review]

**Review by mcafaro — APPROVED** [review]

**Pull request merged**